src/components/Navbar/NavBar.tsx

```diff
@@ -296,10 +318,65 @@ const ReactNavLink = styled(Link)`
     color: black;
   }
   :hover {
-    color: #9e9e9e;
-  }
+    color: #9e9e9e;}
+  position:relative;
 `;
 
+
 const Close = styled(CloseIcon)`
   cursor: pointer;
+`;
+
+
+const Dropdown = styled.div`
+  display:flex;
+  z-index: 11;
+  display: block;
+  position: absolute;
+  left: 145px;
+  top: 19px;
+  width: 213px;
+  height: 183px;  
```

Comment:
I know that we don't do this in our codebase, but we really should. It's general accepted convention to keep margin widths, etc. even numbers. This should be reflected in designs too.

cc @nanxiy what do you think about establishing a "multiple of 8px" rule? or at the very least a multiple of 2px

Reply:
Yes we definitely can do that for our unique designs but for Weebly.. I don't think I have control over that. I would reference our live page for their specs

Reply:
Sounds good! Square follows a standard for multiples of 8px which is why I suggested it, or 2px if you need spacing < 8px. I know some companies that do multiples of 5px — so whatever you think would look good

Reply:
I think Weebly we can just keep it as is